src/test/util/setup_common.cpp

```diff
@@ -95,6 +95,26 @@ std::ostream& operator<<(std::ostream& os, const uint256& num)
     return os;
 }
 
+namespace dash {
```

Comment:
can we please come up with a better name here?

Reply:
Initialization logic isn't limited to LLMQ, masternodes more or less are their own subsystem, even if they interact heavily with everything else, the only thing encompassing them all is that they're Dash-specific.

Reply:
so far as it used only in src/test/util/setup_common.cpp and `validation_chainstatemanager_tests.cpp` (only RegTest) and doesn't used inside init.cpp for normal app initialization, let's give it more specific name? Such as `DashRegnetInit` or `DashTestSetup` or something similar.


Btw, I don't like direct call of `dash::DestroyComponents`

Reply:
Neither do I but the chainstates created affect `m_node`, affecting the scope of the entire function and Dash's subsystems are tied to a chainstate, so if they're torn down and rebuilt, we cannot reuse the objects initialized on the older chainstate.

Naming them `DashTestSetup` and `DashTestSetupClose` works for me. Thoughts @PastaPastaPasta?